### PR TITLE
Make sure update-codegen also updates the config network CM

### PR DESCRIPTION
**Pull request by vagababov**
- drink coffee
- pull new deps
- update the script
- update tools.go
- add some example periods to verify the generation works

/assign mattmoor @tcnghia

### CODE REVIEW

File: config/config-network.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ metadata:
   labels:
     serving.knative.dev/release: devel
   annotations:
-    knative.dev/example-checksum: "3317ccca"
+    knative.dev/example-checksum: "2653766a"
```
Comment (mattmoor) [inline config/config-network.yaml:23]: Just to check: if this isn't updated, will the tests fail?

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve
/hold

Hold for the question

File: config/config-network.yaml (hunk 1 above)
Comment (tcnghia) [reply config/config-network.yaml:23]: Didn't fail in my PR.

Comment (vagababov) [reply config/config-network.yaml:23]: Dunno, do we call verify codegen?

**Comment by vagababov** [comment]
In general it should, since I've added this to the `update-codegen` so it'll generate a diff now.
/unhold

**Pull request merged**